modules/library/metadata/src/main/java/org/geotools/util/GrowableInternationalString.java

```diff
@@ -142,13 +141,11 @@ public synchronized void add(final Locale locale, final String string)
                     }
             }
             String old = localMap.get(locale);
-            if (old != null) {
```

Comment:
So when adding a content for a Locale that already has it, it will be now replaced.
I guess this was done to avoid exception GeoServer side on the UI component. But this is not the desired behaviour, since the client code should be the one making sure to not add duplicated entries to avoid exception to be thrown.
I would put this code back as it was and fix the validation for duplicated entries in the component on the GeoServer related pr.

Reply:
done